### PR TITLE
Fixing loading to account for Gunicorn's multiple instances

### PR DESCRIPTION
**Pull request by ramirezg**
Moved logic out of `wsgi.py` to ` cf.sh`

### CODE REVIEW

File: cf.sh [hunk 1]
```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 echo "------ Starting APP ------"
-gunicorn foia_hub.wsgi:application --log-file -
+if [ $CF_INSTANCE_INDEX = "0" ]; then
+    echo "----- Migrating Database -----"
+    python manage.py migrate --noinput
+    echo "----- Loading Agency Contacts -----"
+    python manage.py load_agency_contacts
+fi
+waitress-serve --port=$VCAP_APP_PORT foia_hub.wsgi:application
```
Comment (khandelwal) [inline cf.sh]: Looks like this pull request doesn't just move the data loading and migrations to cf.sh. It also moves us from gunicorn to waitress. It's fine if that's your recommendation, but you should call that out. If we're going to move back to waitress, we're going to need to run a few more load tests and figure out our failover point.

Comment (ramirezg) [inline cf.sh]: @khandelwal Actually, it should be gunicorn. I accidentally left it in after testing.

**Comment by khandelwal** [comment]
Now that this is merged, I'm going to a do a deploy to staging, and check for errors in the logs.

**Comment by khandelwal** [comment]
This looks good in staging.

**Comment by khandelwal** [comment]
Also, looks like the 2014 times were updated correctly.

**Comment by ramirezg** [comment]
@khandelwal has this been deployed to production?

**Comment by khandelwal** [comment]
Deployed. https://github.com/18F/foia-hub/releases/tag/0.3

**Pull request merged**